server/supernova/server/server.cpp

```diff
@@ -269,27 +269,42 @@ static bool set_realtime_priority(int thread_index)
 #ifdef NOVA_TT_PRIORITY_RT
 
 #ifdef JACK_BACKEND
+#ifndef _WIN32
         int priority = instance->realtime_priority();
         if (priority >= 0)
             success = true;
-
-#elif _WIN32
+#else	// JACK_BACKEND not win32
+		int priority = thread_priority_interval_rt().second;
+		success = true;
+#endif
+#elif _WIN32 // not JACK_BACKEND and _WIN32
         int priority = thread_priority_interval_rt().second;
-#else
+		success = true;
+#else	// not JACK_BACKEND not _WIN32 (should be APPLE portaudio)
```

Comment:
IIUC there are only three conditions here, not 4 - ifdef _WIN32, elif defined(JACK_BACKEND), and else. please reareange

Reply:
No, conditions could even be 6: two audioapis and three operative systems , but as we dont have linux-portaudio or apple-jack there are four in the end linux-jack, windows-portaudio, windows-jack and apple-portaudio

Reply:
right, but as it stands there are only 4, and two of them are exactly the same, so i am suggesting that you combine them for clarity.